Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -97,11 +97,12 @@ public void SetPlayerInput(FPSInputManager playerInput)
         var canvas = hudController.GetComponent<Canvas>();
         canvas.worldCamera = fpsInput.GetComponentInChildren<Camera>();
         canvas.planeDistance = 0.11f;
+        
         // Set player color
-        var meshRenderer = meshBase.GetComponentInChildren<SkinnedMeshRenderer>();
-        var ragdollRenderer = ragdoll.GetComponentInChildren<SkinnedMeshRenderer>();
-        meshRenderer.materials[0].SetColor("_Color", identity.color);
-        ragdollRenderer.materials[0].SetColor("_Color", identity.color);
+        // var meshRenderer = meshBase.GetComponentInChildren<SkinnedMeshRenderer>();
+        // var ragdollRenderer = ragdoll.GetComponentInChildren<SkinnedMeshRenderer>();
+        // meshRenderer.materials[0].SetColor("_Color", identity.color);
+        // ragdollRenderer.materials[0].SetColor("_Color", identity.color);
```

Comment:
We shouldn't disable the only visual difference between our players. Please uncomment this code, since it apparently works :)))))